src/sentry/integrations/slack/actions/notification.py

```diff
@@ -120,12 +129,71 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                     "attachments": json.dumps(attachments),
                 }
 
+            rule = rules[0] if rules else None
+            rule_to_use = self.rule if self.rule else rule
+            rule_id = rule_to_use.id if rule_to_use else None
+            rule_action_uuid = self.data.get("action_uuid", None)
+            if not rule_action_uuid:
+                # We are logging because this should never happen, all actions should have an uuid
+                # We can monitor for this, and create an alert if this ever appears
+                _default_logger.info(
```

Comment:
I see here and there that we're adding a lot of good logs 🎉 
Are these documented somewhere and have alerts assigned to them? How do we plan to monitor them?

Reply:
I'm not sure where we document our logs, but that would actually be great if we did! Would make debugging easier too so we can just go to the doc and see which logs to look for - or better yet, a link that has the query with all the logs relevant!